OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -192,6 +183,23 @@ void INotifyCreated.Created(Actor self)
 				if (!string.IsNullOrEmpty(Info.LoadedCondition))
 					loadedTokens.Push(conditionManager.GrantCondition(self, Info.LoadedCondition));
 			}
+
+			// Defer notifications until we are certain all traits on the cargo are initialised
```

Comment:
The problem here isn't with the cargo, but with `INotifyPassengerEntered` on the transport. Can we tweak this comment to make this clear?